pyproject.toml

```diff
@@ -146,25 +146,22 @@ ECM_Example = "pybamm.input.parameters.ecm.example_set:get_parameter_values"
 MSMR_Example = "pybamm.input.parameters.lithium_ion.MSMR_example_set:get_parameter_values"
 Chayambuka2022 = "pybamm.input.parameters.sodium_ion.Chayambuka2022:get_parameter_values"
 
-[tool.setuptools]
-include-package-data = true
+[tool.hatch.build]
+packages = ["src/pybamm"]
+include = [
+    "src/pybamm/**/*.txt",
+    "src/pybamm/**/*.md",
+    "src/pybamm/**/*.csv",
+    "src/pybamm/**/*.py",
+    "src/pybamm/CITATIONS.bib",
+    "src/pybamm/plotting/mplstyle",
```

Comment:
The comment explaining why this is not automatically generated is left below when the code was moved.

It would be nice if we could switch to this being automatic though

Reply:
Makes sense, it's automatic for setuptools if `include_package_data` is set to `True`. Could you check if a similar option is available for hatchling, @vidipsingh?

Reply:
It is _not_ automatic for setuptools (you need to specify files in` MANIFEST.in`, as we do right now), but it is automatic for hatch 😉

See my comment.